lib/sparse/src/index/search_context.rs

```diff
@@ -258,6 +253,17 @@ impl<'a, 'b, T: PostingListIter> SearchContext<'a, 'b, T> {
         if self.postings_iterators.is_empty() {
             return Vec::new();
         }
+
+        {
+            // Measure CPU usage of indexed sparse search.
+            // Assume the complexity of the search as total volume of the posting lists
+            // that are traversed in the batched search.
+            let cpu_counter = self.hardware_counter.cpu_counter_mut();
+            for posting in self.postings_iterators.iter() {
+                cpu_counter.incr_delta_mut(posting.posting_list_iterator.len_to_end());
+            }
```

Comment:
The original does count on `for_each_till_id(batch_last_id, ...)`. The new implementation counts `.len_to_end()`. Are they the same?

Reply:
According to tests, yes. But overall I think it is correct to estimate search complexity as a sum of posting length